pkg/cloudprovider/drift.go

```diff
@@ -38,6 +42,7 @@ import (
 const (
 	K8sVersionDrift   cloudprovider.DriftReason = "K8sVersionDrift"
 	ImageVersionDrift cloudprovider.DriftReason = "ImageVersionDrift"
+	SubnetDrift       cloudprovider.DriftReason = "SubnetDrift"
```

Comment:
Not to do with your PR, but wondering if we want to add an aks prefix to these to distinguish them as cloud provider drift reasons.

Reply:
Maybe thats something that belongs inside the actual core code since eventual controller will take a list of cloudprovider reasons in the future?